### ISSUE TITLE
Regenerate tests file causes an exception

### ISSUE THREAD

**Issue by marbel82** [RandomEngy/UnitTestBoilerplateGenerator#27]
I'm using:
    Visual Studio Community 2022 (64-bit) - Preview 
    Version 17.2.0 Preview 1.0

First, I generated a tests file by UnitTestBoilerplateGenerator.

Later I added a new method and try to generate the tests file another time, and  I got an exception:

```
System.InvalidOperationException: Test file already exists.

   at UnitTestBoilerplate.Services.TestGenerationService.<GenerateUnitTestFileAsync>d__6.MoveNext() in C:\projects\unittestboilerplategenerator\src\Services\TestGenerationService.cs:line 0

--- End of stack trace from previous location where exception was thrown ---

   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

   at UnitTestBoilerplate.ViewModel.CreateUnitTestBoilerplateViewModel.<CreateUnitTestAsync>d__58.MoveNext() in C:\projects\unittestboilerplategenerator\src\ViewModel\CreateUnitTestBoilerplateViewModel.cs:line 162

--- End of stack trace from previous location where exception was thrown ---

   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

   at UnitTestBoilerplate.ViewModel.CreateUnitTestBoilerplateViewModel.<<get_CreateUnitTestCommand>b__57_0>d.MoveNext() in C:\projects\unittestboilerplategenerator\src\ViewModel\CreateUnitTestBoilerplateViewModel.cs:line 147

```

**Comment by RandomEngy** [comment]
Regenerating tests is not supported. The extension works on a template system and does not know how to deal with possible edits that may have been made since the file was generated.

**Comment by GlennPiper** [comment]
@RandomEngy - Could it at least be updated to notify the user and prompt for a new test class name or something similar so that the developer could then pull out what they desire?

**Comment by RandomEngy** [comment]
@GlennPiper I just released 2.7.2, which has a dialog to allow to to pick how to resolve the file conflict: Overwrite, Use new name or Cancel.
